src/test/java/minkowitz/dictionary/DictionaryTest.java

```diff
@@ -0,0 +1,155 @@
+package minkowitz.dictionary;
+
+import org.junit.Test;
+
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+
+import static org.junit.Assert.assertEquals;
+
+public class DictionaryTest {
+
+    @Test
+    public void containsTrue() throws FileNotFoundException
+    {
+        //given
+        Dictionary dictionary = new Dictionary();
+        //when
+        //then
+        assertEquals(true, dictionary.contains("AA"));
```

Comment:
Instead of doing `assertEquals(true)` and `assertEquals(false)`
use 
`assertTrue()` and `assertFalse()`